examples/bri-3/src/bri/identity/bpiAccounts/api/accounts.controller.spec.ts

```diff
@@ -24,19 +23,31 @@ import { MockBpiSubjectStorageAgent } from '../../bpiSubjects/agents/mockBpiSubj
 import { BpiSubject } from '../../bpiSubjects/models/bpiSubject';
 import { BpiSubjectType } from '../../bpiSubjects/models/bpiSubjectType.enum';
 import { BpiSubjectAccount } from '../../bpiSubjectAccounts/models/bpiSubjectAccount';
+import { AccountsProfile } from '../subject.accounts.profile';
+import { Mapper } from '@automapper/core';
+import { AutomapperModule } from '@automapper/nestjs';
+import { classes } from '@automapper/classes';
+import { SubjectsProfile } from '../../bpiSubjects/subjects.profile';
+import { SubjectAccountsProfile } from '../../bpiSubjectAccounts/subjectAccounts.profile';
 
 describe('AccountController', () => {
   let accountController: AccountController;
   let mockBpiSubjectStorageAgent: MockBpiSubjectStorageAgent;
   let mockBpiSubjectAccountsStorageAgent: MockBpiSubjectAccountsStorageAgent;
+  let mapper: Mapper;
 
   beforeEach(async () => {
-    mockBpiSubjectStorageAgent = new MockBpiSubjectStorageAgent(new Mapper());
+    mockBpiSubjectStorageAgent = new MockBpiSubjectStorageAgent(mapper);
     mockBpiSubjectAccountsStorageAgent = new MockBpiSubjectAccountsStorageAgent(
-      new Mapper(),
+      mapper,
```

Comment:
same comment, undefined passed here, probably not needed to inject mapper at all in mock agent